spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cassandra/CassandraProperties.java

```diff
@@ -117,6 +117,53 @@
 	 */
 	private boolean ssl = false;
 
+	/**
+	 * Socket option: TCP keepalive.
+	 */
+	private Boolean keepAlive;
+
+	/**
+	 * Pooling option: core number of connections per local host.
+	 */
+	private Integer corePoolLocal;
+
+	/**
+	 * Pooling option: maximum number of connections per local host.
+	 */
+	private Integer maxPoolLocal;
+
+	/**
+	 * Pooling option: core number of connections per remote host.
+	 */
+	private Integer corePoolRemote;
+
+	/**
+	 * Pooling option: maximum number of connections per remote host.
+	 */
+	private Integer maxPoolRemote;
+
+	/**
+	 * Pooling option: threshold that triggers the creation of a new connection to a local
+	 * host.
+	 */
+	private Integer newConnectionThresholdLocal;
+
+	/**
+	 * Pooling option: threshold that triggers the creation of a new connection to a
+	 * remote host.
+	 */
+	private Integer newConnectionThresholdRemote;
+
+	/**
+	 * Pooling option: maximum number of requests per local connection.
+	 */
+	private Integer maxRequestsPerConnectionLocal;
+
+	/**
+	 * Pooling option: maximum number of requests per remote connection.
+	 */
+	private Integer maxRequestsPerConnectionRemote;
+
```

Comment:
It looks like we have a `local` and `remote` split for configuration so it would be nice to have that reflected in the namespace as well. How about a `spring.data.cassandra.local` and `spring.data.cassandra.remote`. We also tend to avoid lengthy property names so maybe there's room for improvements there.